dockerfile_parse/parser.py

```diff
@@ -290,7 +297,7 @@ def _instruction_getter(self, name, env_replace):
         if name != 'LABEL' and name != 'ENV':
             raise ValueError("Unsupported instruction '%s'", name)
         instructions = {}
-        envs = {}
+        envs = self.parent_env
```

Comment:
@lcarva Turns out I was wrong on how the env var was being used here, it's handled correctly in the latest commit